README.md

```diff
@@ -107,8 +107,7 @@ $ /usr/local/bin/master-restart api
 $ /usr/local/bin/master-restart controllers
 ```
 
-
-### Openshift 4
+### Openshift 4.3+ Manual Configuration
```

Comment:
All of the manual configuration could go under a larger heading. 

Also, if our operator does this, I'd argue we don't need to include the manual steps on our front-page README. User only needs to know about it if they're going to have to tweak it.

---

## Manual CORS Configuration

Manual CORs config is needed on these versions:
 - OpenShift x.x
 - OpenShift x.y

### OpenShift 4.2
[... config steps here ...]

### OpenShift 4.3
[... config steps here ...]

### OpenShift 4.4
[... config steps here ...]

Reply:
I made them sub sections like you suggested. I considered removing the section altogether or putting it in another file. I'm a little torn on this since we have people expressing interest in 3 -> 3 migrations and it will apparently remain relevant in current versions after talking to @eriknelson.

Reply:
What do you guys think is best?

Reply:
My vote would be moving CORs config info to another file and linking to that file from here.